desafio-8/desafio-8.html

```diff
@@ -0,0 +1,44 @@
+<html>
+   <head>
+      <title>Formulário</title>
+      <link rel="stylesheet" href="desafio-8.css">
+   </head>
+   <body>
+      <header>
+         <img scr="https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pngwing.com%2Fpt%2Ffree-png-sppmm&psig=AOvVaw2uYMi17rHg4DRGkJ8jbNoK&ust=1644627532953000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCIjpg5W59vUCFQAAAAAdAAAAABAJ" />
+         <h1>Desafio 8 - Formulário</h1>
+      </header>
+      <main>
+         <form>
+            <label class="nomesobrenome">Nome Completo</label>
+            <input type="text" id="nomesobrenome" class="input-padrao">
+            <label for="email">Email</label>
+            <input type="text" id="email" class="input-padrao">
+            <label for="telefone">Telefone</label>
+            <input type="text" id="telefone" class="input-padrao">
+            <input type="submit" value="Enviar formulário" class="button">
```

Comment:
in my screen resolution (mac), is not possible to see this button

<img width="1435" alt="image" src="https://user-images.githubusercontent.com/38358239/153593115-52d419e5-c37e-4cd1-9662-93e84cf52e4d.png">

Reply:
on mac still doesn't work, the footer is above the form :( 

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/38358239/153868630-0f8a59ba-14e4-4be2-b962-221a3a855fd0.png">

Reply:
Rai, I set the height in main and now it has the scroll:

https://user-images.githubusercontent.com/42494406/154482239-8d5816b2-223e-4808-a7ca-0373a5d1680e.mov